src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -54,13 +93,15 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   const containerCssVars = useCss({
     "--event--margin-left": `${eventMarginLeftPx}px`,
     "--event--width": `${eventWidthPx}px`,
+    "--event--widen": `${expandedEventPx}px`,
   });
 
   const containerClasses = classNames(
     "ScheduleEvent",
     {
       "ScheduleEvent--live": isEventLive(event),
       "ScheduleEvent--users": isPersonalizedEvent,
+      "ScheduleEvent--wide": isExpanded || isExpandedStatic,
```

Comment:
Since we use the term 'expand' / 'expanded' everywhere else, why would we not use it here as well?

What's the difference between `isExpanded` and `isExpandedStatic`?

Reply:
described the difference here https://github.com/sparkletown/sparkle/pull/1733#discussion_r665959998
gonna rename it tho

Reply:
f4e3df2 done